src/library/baseplaylistfeature.cpp

```diff
@@ -350,17 +358,18 @@ void BasePlaylistFeature::slotDeletePlaylist() {
 
 void BasePlaylistFeature::slotImportPlaylist() {
     //qDebug() << "slotImportPlaylist() row:" << m_lastRightClickedIndex.data();
-    QString playlist_file = getPlaylistFile();
-    if (playlist_file.isEmpty()) {
+    const QString playlistFile = getPlaylistFile();
+    if (playlistFile.isEmpty()) {
         return;
     }
 
     // Update the import/export playlist directory
-    QFileInfo fileName(playlist_file);
-    m_pConfig->set(ConfigKey("[Library]", "LastImportExportPlaylistDirectory"),
-            ConfigValue(fileName.dir().absolutePath()));
+    QString fileDirectory(playlistFile);
+    fileDirectory.truncate(playlistFile.lastIndexOf(QDir::separator()));
```

Comment:
I suspect this always truncates at position 0, see https://bugs.launchpad.net/bugs/1964508

Native separator is `\` on Windows and maybe Qt has already converted the path to use `/`? In that case `lastIndexOf("\")` would return -1 which is equivalent to passing 0 to `truncate`
https://doc.qt.io/qt-5/qstring.html#truncate
It is not recommended to use QDir::separator() for constructing paths, but _appearantly_ it also creates issues when computing paths
https://doc.qt.io/qt-5/qdir.html#separator